jerry-core/vm/vm-utils.c

```diff
@@ -59,15 +59,42 @@ vm_is_direct_eval_form_call (void)
  * @return array ecma value
  */
 ecma_value_t
-vm_get_backtrace (uint32_t max_depth) /**< maximum backtrace depth, 0 = unlimited */
+vm_get_backtrace (uint32_t max_depth, /**< maximum backtrace depth, 0 = unlimited */
+                  ecma_object_t *ignored_function_p) /**< ignore functions up to this function */
 {
 #if ENABLED (JERRY_LINE_INFO)
+  vm_frame_ctx_t *context_p = JERRY_CONTEXT (vm_top_context_p);
+
```

Comment:
I'd put here an assert, just in case if someone tries to call it from internal code rather than the new API function in the future. Something like: 
```
ecma_object_type_t type = ecma_get_object_type (ignored_function_p);
JERRY_ASSERT (type == ECMA_OBJECT_TYPE_FUNCTION || type == ECMA_OBJECT_TYPE_NATIVE_FUNCTION || type == ECMA_OBJECT_TYPE_BOUND_FUNCTION);
```

Reply:
The `ignored_function_p` is not mandatory, it can be NULL.

Reply:
I have added an assertion below.